index.js

```diff
@@ -5,14 +5,50 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "green"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+let post = document.querySelector('.traveler')
+let photo = post.querySelector('img')
+let newName = post.querySelector('h2')
+let newNickName = post.querySelector('em')
+
+photo.src = traveler.photo
+newName.textContent = traveler.name
+newNickName.textContent = traveler.nickname
+
 
 
 /***** Deliverable 4 *****/
 
+const ul = document.querySelector('ul#animals')
```

Comment:
Smart idea writing this outside of the callback function for `forEach`. Saves computational time & energy!